File: prob6/prob6.js
var rand_print = function (rand_num){
	var greaterThan = []
	var lessThan = []
	if (rand_num > 40) {
		for (var i = 40; i <= rand_num; i++) {
			if (i % 2 !== 0){
				greaterThan.push(i)
			}
		}
		document.write(greaterThan.reverse())
	} else {
		for (var i = rand_num; i < 40; i++) {
			if (i % 2 !== 0) {
				lessThan.push(i)
			}
		}
		document.write(lessThan.reverse())
	}
}

var rand_num = (Math.floor((Math.random() * 100)))
document.write("The number is " + rand_num + "<br/>")
rand_print(rand_num)

File: prob6/prob6.py
from random import randrange

greater_than = []
less_than = []

def rand_print(num):
	if num > 40:
		for x in range(40,num+1):
			if x % 2 != 0:
				greater_than.append(x)
		reverse(greater_than)
	else:
		for x in range(0,num+1):
			if x % 2 != 0:
				less_than.append(x)
		reverse(less_than)

def reverse(lst):
	for i in reversed(lst):
		print i


rand_num = randrange(100)
print "The number is " + str(rand_num)
print ""
rand_print(rand_num)